src/scripts/main.js

```diff
@@ -18,10 +19,62 @@ const food = {
   },
 };
 
-const tree = document.querySelector('#tree');
+// const tree = document.querySelector('#tree');
+
+// function createTree(element, data) {
+//   if (!Object.keys(data).length) {
+//     return;
+//   }
+
+//   const ul = document.createElement('ul');
+
+//   for (const key in data) {
+//     if (data.hasOwnProperty(key)) {
+//       const li = document.createElement('li');
+
+//       li.textContent = data;
```

Comment:
In the commented-out code, `li.textContent = data;` is incorrect. It should be `li.textContent = key;` to properly set the text content of the list item to the current key.